mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <!-- App name for private browsing mode, only the "(Private)" portion should be localized. -->
-    <string name="app_name_private">Firefox Preview (Private)</string>
+    <string name="app_name_private">Private Firefox Preview</string>
```

Comment:
This needs a string ID update if localizers are to be made aware they need to update this string